src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -11,11 +11,16 @@
 import org.springframework.web.bind.annotation.RequestMapping;
```

Comment:
For each Java class, please provide:
- class header
- Javadoc comment for each methods